charts/minecraft-bedrock/templates/minecraft-svc.yaml

```diff
@@ -7,8 +7,12 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  {{- if .Values.serviceAnnotations }}
   annotations:
-    {{ toYaml .Values.serviceAnnotations | indent 4 }}
+    {{- range $key, $value := .Values.serviceAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```

Comment:
I believe the other way to fix the extra indentation was to apply chomping to the left

```
    {{- toYaml .Values.serviceAnnotations | indent 4 }}
```

...however, your change is more consistent with similar sections in the templates, so the current change is best.